libs/api/domains/signature-collection/src/lib/dto/owner.input.ts

```diff
@@ -11,11 +11,11 @@ export class SignatureCollectionOwnerInput {
   @IsString()
   name!: string
 
-  @Field()
+  @Field({ nullable: true })
   @IsString()
-  phone?: string
+  phone!: string
```

Comment:
Shouldn't it be `?` instead of `!` when `nullable: true`?

Reply:
removed nullable